pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -14,6 +14,33 @@ import (
 // HyperConvergedName is the name of the HyperConverged resource that will be reconciled
 const HyperConvergedName = "kubevirt-hyperconverged"
 
+// HyperConverged is the Schema for the hyperconvergeds API
+// +k8s:openapi-gen=true
+type HyperConvergedConfig struct {
+
+	// nodeSelector is the simplest recommended form of node selection constraint:
+	// It specifies a map of key-value pairs. For the pod to be eligible to run on a node,
+	// the node must have each of the indicated key-value pairs as labels
+	// (it can have additional labels as well).
+	// See https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+	// for additional details.
+	// InfraNodeSelector will be applied to all the infra pods of the components
+	// managed by the HyperConverged Cluster Operator
+	// +optional
+	InfraNodeSelector corev1.NodeSelector `json:"infraNodeSelector,omitempty"`
```

Comment:
No surprises here with the node selectors. This aligns with what we all discussed a few days ago.

Do we need `infraTolerations` and `workloadTolerations` fields as well here? It seems like we would, since I'd expect their might be use cases where taints/tolerations are used to ensure only kubevirt workloads run on certain nodes. in that situation a user might taint a node and want to give the toleration only to hco related workload components.

Reply:
yes, I think it's a good idea